ArduCopter/mode_loiter.cpp

```diff
@@ -125,17 +125,21 @@ void ModeLoiter::run()
 
     case AltHoldModeState::MotorStopped:
         attitude_control->reset_rate_controller_I_terms();
-        attitude_control->reset_yaw_target_and_rate();
+        attitude_control->reset_yaw_target_and_rate(true);
         pos_control->relax_z_controller(0.0f);   // forces throttle output to decay to zero
         loiter_nav->init_target();
         attitude_control->input_thrust_vector_rate_heading(loiter_nav->get_thrust_vector(), target_yaw_rate, false);
         break;
 
     case AltHoldModeState::Landed_Ground_Idle:
-        attitude_control->reset_yaw_target_and_rate();
-        FALLTHROUGH;
+        attitude_control->reset_yaw_target_and_rate(false);
```

Comment:
Why wouldn't we want to reset_rate on landed ground idle?

Reply:
This provides some damping to the tail if it gets pushed by a gust or something but will stay close to zero as long as we keep the integrator zero.